razorpay-sdk/src/Utility.php

```diff
@@ -9,10 +9,20 @@ class Utility
     public function verifyPaymentSignature($attributes)
     {
         $expectedSignature = $attributes['razorpay_signature'];
-        $orderId = $attributes['razorpay_order_id'];
         $paymentId = $attributes['razorpay_payment_id'];
 
-        $payload = $orderId . '|' . $paymentId;
+        if (isset($attributes['razorpay_order_id']) === true)
+        {
+            $orderId = $attributes['razorpay_order_id'];
+
+            $payload = $orderId . '|' . $paymentId;
+        }
+        else
```

Comment:
@ccrims0n We'll change the SDK too. But this will likely go live before the SDK change.